torchvision/csrc/io/image/image.cpp

```diff
@@ -23,11 +23,11 @@ static auto registry =
     torch::RegisterOperators()
         .op("image::decode_png", &decode_png)
         .op("image::encode_png", &encode_png)
-        .op("image::decode_jpeg", &decode_jpeg)
+        .op("image::decode_jpeg(Tensor data, int mode, bool apply_exif_orientation=False) -> Tensor", &decode_jpeg)
```

Comment:
This helps to make the op BC compatible, otherwise `apply_exif_orientation` becomes required and thus break the old code